### PR TITLE
Masking activity

### PR DESCRIPTION
**Pull request by kiembully**
jamesbond

## https://codepen.io/Kim-Klinton-Vale/pen/XWQVNgN
* ⌛ You have 15 minutes to implement this, take your time (but not too much)!
* 💅 Feel free to embellish and make it pretty!
* 📦 Some basic CSS is provided for you. There are some `TODO` comments in there to help you out.
* 🧙 You can search online if you get stuck.
* 🎉 When you're done, `Fork` 🍴 the Codepen page and share with us the URL.

<p align='center'>* * *</p>

## Tips and hints

- Gradial gradient is used as mask image.
- Use `--x` and `--y` as variables for the horizontal and vertical positions of the radial gradient circle in order for the JS to work.

### CODE REVIEW

File: src/content/masking.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+slug: '/template'
```
Comment (raaims) [inline src/content/masking.md]: rename this to `masking` or into something fun (`james bond`)

Comment (kiembully) [reply src/content/masking.md]: done

**Comment by raaims** [comment]
https://codepen.io/raaims/pen/poBKjMy

**Comment by viancamaulion** [comment]
https://codepen.io/viamaulion/pen/vYMrLYQ

**Comment by notco** [comment]
https://codepen.io/notco/pen/RwOJWzq

**Comment by xinjo21** [comment]
https://codepen.io/El-Razin-Jo/pen/vYMrLBQ

**Comment by janpatrickpalacio** [comment]
https://codepen.io/janpatrick-dev/pen/gOyKaNE

**Comment by redlpd** [comment]
https://codepen.io/redlpd/pen/YzMvymJ

**Pull request merged**